File: 1. Basic Statistics_Level 1/Q9.py
# importing necessary libraries
import pandas as pd
import numpy as np
from scipy.stats import skew
from scipy.stats import kurtosis

Q9_a = pd.read_csv('Q9_a.csv')
Q9_a = Q9_a.iloc[:, 1:3]
# print(Q9_a.head())

for (ColumnName, ColumnData) in Q9_a.iteritems():
	print("*** Measure of Skewness, Kurtosis '{}' ***".format(ColumnName))
	print("Skewness: {:0.3f}".format(skew(ColumnData.values)))
	print("Kurtosis: {:0.3f}".format(kurtosis(ColumnData.values))+'\n')


Q9_b = pd.read_csv('Q9_b.csv')
Q9_b = Q9_b.iloc[:, 1:3]
# print(Q9_b.head())

for (ColumnName, ColumnData) in Q9_b.iteritems():
	print("*** Measure of Skewness, Kurtosis '{}' ***".format(ColumnName))
	print("Skewness: {:0.3f}".format(skew(ColumnData.values)))
	print("Kurtosis: {:0.3f}".format(kurtosis(ColumnData.values))+'\n')

File: 1. Basic Statistics_Level 1/Q9.R
###Q9### Calculating Skewness and Kurtosis
Q9_a <- read.csv(file.choose())
View(Q9_a)
attach(Q9_a)
library(e1071) 
skewness(speed)
kurtosis(speed)
skewness(dist)
kurtosis(dist)

Q9_b <- read.csv(file.choose())
View(Q9_b)
attach(Q9_b)

skewness(SP)
kurtosis(SP)
skewness(WT)
kurtosis(WT)